### DISCUSSION TITLE
Should I have one single prisma client instance in my whole application?

### DISCUSSION THREAD

**Discussion by dbalatoni13** [prisma/prisma#6391] [Q&A]
I am new to next.js and prisma. Is it okay to declare prisma client in every file or should I have only one instance?

**Comment by dbalatoni13** [comment]
I found [this helpful link](https://www.prisma.io/docs/support/help-articles/nextjs-prisma-client-dev-practices).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
